app/assets/stylesheets/index.scss

```diff
@@ -0,0 +1,63 @@
+
+$font-color: #088da5;
+
+body {
+
+  background-color: black;
+
+}
+
+h1,
+h3 {
+  text-align: center;
+  color: $font-color;
+
+}
+
+
+#about {
+
+  border: 3px solid #000000;
+  text-align: center;
+  padding: 5px;
+  margin: 10px;
+  color: $font-color;
+
+}
+
+#recently-reviewed {
+
+  border: 3px solid #000000;
```

Comment:
Color literals like `#000000` should only be used in variable declarations; they should be referred to via variable everywhere else.